### PR TITLE
Add test test_from_dockerfile()

### PR DESCRIPTION
**Pull request by michal-josef-spacek**
Similar test as in 5.30

### CODE REVIEW

**Comment by zmiklank** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

File: examples/from-dockerfile/5.30-mod_fcgid/Dockerfile.s2i [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ubi8/perl-530
+
+# Add application sources to a directory that the assemble script expects them
+# and set permissions so that the container runs without root access
+USER 0
+ADD app-src /tmp/src
```
Comment (phracek) [inline examples/from-dockerfile/5.30-mod_fcgid/Dockerfile.s2i:6]: Sorry, I am not an expert in mod_fcgid, but
`Plack as an FCGI server` is not needed for `Dockerfile.s2i`?

**Review by phracek — COMMENTED** [review]
Thanks for the PR!. One comment from my point of view. Let's see, what will happened with tests.

File: examples/from-dockerfile/5.30-mod_fcgid/Dockerfile.s2i (hunk 1 above)
Comment (michal-josef-spacek) [reply examples/from-dockerfile/5.30-mod_fcgid/Dockerfile.s2i:6]: s2i perl container is able to run *.psgi app. There are possibilities:
* Autodetect *psgi in app root (this is case in this test)
* Set application script by PSGI_FILE
Application in test is https://github.com/sclorg/dancer-ex.git, which has app.psgi script, which will be run.

**Comment by phracek** [comment]
@michal-josef-spacek RHEL8 failure:

```bash
[1m[INFO] Check building using a Dockerfile...[0m

ERROR: Dockerfile /tmp/sclorg/s2i-perl-container/5.26-mod_fcgid/test/examples/from-dockerfile/5.26-mod_fcgid/Dockerfile does not exist or is empty.
Terminating the Dockerfile build.
RESULT: 1
```

**Comment by phracek** [comment]
[test-all]

**Comment by michal-josef-spacek** [comment]
[test-all]

**Comment by zmiklank** [comment]
The test env for RHEL8 should be fixed now, rerunning the tests:
[test-all]

**Comment by michal-josef-spacek** [comment]
@phracek, @zmiklank Do you know what happened in "Testing Farm - RHEL7 — Build finished" failure?

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
RHEL7 failed with this reason:

```bash
Could not resolve host: rhsm-pulp.corp.redhat.com; Unknown error\"\nTrying other mirror.\n\n\nError downloading
```

I will reschedule this test as soon as the rest are finished.

**Comment by michal-josef-spacek** [comment]
[test]

**Comment by michal-josef-spacek** [comment]
[test-all]

**Comment by michal-josef-spacek** [comment]
@phracek I added support for 5.34.

**Review by phracek — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**